pkgs/development/python-modules/slither/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  prettytable,
+  pythonOlder,
+}:
+
+buildPythonPackage rec {
+  pname = "slither-analyzer";
```

Comment:
Why `slither-analyzer` and not `slither`.

Reply:
In that case I think the drv name should stay as it is, and the attribute name should also be `slither-analyzer`.
Upstream should possibly change their name to that proper.